less/mixins.less

```diff
@@ -1 +1,30 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.flex {
+    display: flex;
+}
+
+.row {
+    flex-flow: row wrap;
+}
+.col {
+    flex-flow: column wrap;
+}
+
+.btn(@wid, @bord) {
+    color: white;
+    background-color: @btn-bg-color;
+    padding-top: 1.2%;
+    padding-bottom: 0.9%;
+    padding-left: 3.5%;
+    width: @wid;
+    border-radius: @bord;
+    border-color: @btn-bg-color;
+
+    @media @mobile {
+        width: @wid;
+        .flex;
+        flex-flow: row wrap;
+        justify-content: center;
+        border-radius: @bord;
+    }
+}
```

Comment:
- Navigation Styles: Use the navigation.less file for styling. - DONE!
- Main Content Styles: Use the home-page.less file for styling - DONE!
- LESS Mixins: Create and use 2 different mixins to aid your styling. Use the mixins.less file for your mixins - DONE!
- LESS Parametric Mixin: create a parametric mixin that is used to create the sign up button styles. - DONE!
-  Use at least 2 parameters to create your button - DONE!
-  Create a hover state that swaps the background color and font color of the base button styles. - DONE!